### PR TITLE
docs: Add first draft

### PR DESCRIPTION
**Pull request by Voziv**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ratehub/code-style/blob/master/CONTRIBUTING.md#commit
- [ ] Changes have been tested (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Separate commits have been made for each service (for deploy actions)
- [ ] Architecture diagrams have been updated

## PR Type
What kind of change does this PR introduce?

- [ ] Deployment (faas or argocd app)
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
No generalized documentation for contributing

Jira issue: PLAT-1363

## What is the new behavior?
A central place to keep commit guidelines

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information
If we can adopt a generalized contributing MD file then individual repos can point here and only mention what's changed (which in most cases I think will be very little, just things like a list of scopes)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+5. Follow our [Coding Rules](#rules).
+
+6. Commit your changes using a descriptive commit message that follows our [commit message conventions](#commit).
+   Adherence to these conventions is necessary because release notes are automatically generated from these messages.
+
+     ```shell
+     git commit --all
+     ```
+   Note: the optional commit `-a` command line option will automatically "add" and "rm" edited files.
```
Comment (drew887) [inline CONTRIBUTING.md:68]: **[BREAKING]** I feel this is a bit confusing since it shows --all but then talks about -a

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+##### Scope
+The scope should be the name of one of the root folders.
+
+Note: DO NOT USE JIRA tickets as scopes! Either use the footer or place them
+at the end of the subject
+
+The following is the list of supported scopes:
+
+Please see each individual repository for a list of acceptable scopes!
```
Comment (drew887) [inline CONTRIBUTING.md]: **[BREAKING]** When did we change our stance on this? Last I checked the final word was from CR [here](https://ratehub.slack.com/archives/C507X0LLD/p1617853147112800?thread_ts=1615840771.098800&cid=C507X0LLD)

 ```
Final word on commit convention and (scope) in particular is... do what you want.

The JIRA issue must be noted in the commit message somewhere. 
I prefer to use scope for 'area of codebase' but unless it ends up causing process issues I don't feel strongly enough to mandate something here. 
Also note that scope is optional so you can leave out the (...) but not the : 
```

Imo this should just read `Please see each individual repository for a list of acceptable scopes!` to be safe :+1:

**Review by drew887 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Voziv) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (Voziv) [reply CONTRIBUTING.md]: Already changed, was a leftover from migrating this doc from ratehub-k8s 🗡️ 

Definitely per repo for scopes

**Review by drew887 — APPROVED** [review]

**Review by nmorey-ratehub — APPROVED** [review]

**Pull request closed without merging**